### PR TITLE
Add "etcd_operation" and "etcd_type" variables to Grafana dashboard

### PR DESCRIPTION
**Pull request by oxddr**
This should ease querying and allows to remove some dedicated graphs as they can be now easily substituted.

### CODE REVIEW

File: clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py [hunk 1]
```diff
@@ -29,7 +29,24 @@ class Dashboard(g.Dashboard):
         default=g.Templating(
             list=[
                 # Make it possible to use $source as a source.
-                g.Template(name="source", type="datasource", query="prometheus")
+                g.Template(name="source", type="datasource", query="prometheus"),
+                g.Template(
```
Comment (mborsz) [inline clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py]: I think this adds variables to all dashboards including the ones which has nothing to do with etcd (e.g. dns, network). Is this correct?

Can we somehow avoid that?

Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py]: Done.

File: clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py [hunk 2]
```diff
@@ -21,18 +21,14 @@
...
     time = attr.ib(default=g.Time("now-30d", "now"))
-    templating = attr.ib(
-        default=g.Templating(
-            list=[
-                # Make it possible to use $source as a source.
-                g.Template(name="source", type="datasource", query="prometheus")
-            ]
-        )
-    )
+    # Make it possible to use $source as a source.
```
Comment (mborsz) [inline clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py]: IIUC you dropped new variables completely.

I didn't mean to remove etcd_operation and etcd_type variables completely -- rather to somehow move them to master-dashboard only.

Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py]: They were moved not deleted, please see bottom of master-dashboard.dashboard.py

**Comment by mborsz** [comment]
Cool! Thanks!

/lgtm

**Pull request merged**